docs/source/apidoc.rst

```diff
@@ -62,6 +62,13 @@ Task
 .. autoclass:: radical.pilot.Task
    :members:
 
+Raptor
+------
+.. autoclass:: radical.pilot.raptor.Master
+   :members:
+.. autoclass:: radical.pilot.raptor.Worker
+   :members:
+
```

Comment:
Note that `Pilot.submit_tasks()` and `Pilot.submit_raptors()` is not documented (as of RP 1.34, at least)

```suggestion

A `radical.pilot.Task` managing a `radical.pilot.raptor.Master` instance is created using
``mode=rp.RAPTOR_MASTER``, or through :py:func:`~radical.pilot.Pilot.submit_raptors()`.
The object returned to the client is a `Task` subclass with additional features.

.. autoclass:: radical.pilot.raptor_tasks.Raptor
    :members:
```